crates/indexer/src/models/v2_objects.rs

```diff
@@ -157,4 +152,44 @@ impl Object {
             Ok(None)
         }
     }
+
+    /// This is actually not great because object owner can change. The best we can do now though
+    fn get_object_owner(
```

Comment:
I wonder if this can lead to some performance/failure issue for the processor if the data is stored in remote databases (e.g. in Hive). If a processor has to query the object owner multiple times, this means multiple remote calls

Reply:
We don't delete that often so it should be fine for now.